presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaMetadata.java

```diff
@@ -186,8 +185,7 @@ public ConnectorTableMetadata getTableMetadata(ConnectorSession session, Connect
                 columns.put(tableName, getTableMetadata(tableName).getColumns());
             }
             catch (TableNotFoundException e) {
-                // Normally it would mean the table disappeared during listing operation
-                throw new IllegalStateException(format("Table %s cannot be gone because tables are statically defined", tableName), e);
+                // information_schema table
```

Comment:
ah

Reply:
Can you assert that it's really `information_schema` table?

Reply:
i can.
it can also be a system table (should the connector expose one)